{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/test_settings.py

```diff
@@ -0,0 +1,8 @@
+from {{ cookiecutter.project_slug }}.settings import *
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": os.path.join(BASE_DIR, "test_db.sqlite3"),
+    }
+}
```

Comment:
This is probably a fine default, but won't work if we use Postgres-only fields like JSONField, which is a pretty common need. Maybe we can add a comment with a sample config for postgres?

Reply:
Yea an in-memory lightweight DB like sqlite is generally preferred for backend tests that need a DB (Django-specific in our case). I could definitely see situations where a project would need something different. My current plan was to cross that bridge when we come to it and maybe leave it to the app owner to worry about. The bootstrapper is a good starting point, but can't cover all edge cases.

Reply:
Cool, I'm fine with rolling with this and if it causes frequent issues, then we can revisit.